staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go

```diff
@@ -74,12 +82,26 @@ func (m *PathRecorderMux) trackCallers(path string) {
 // refreshMuxLocked creates a new mux and must be called while locked.  Otherwise the view of handlers may
 // not be consistent
 func (m *PathRecorderMux) refreshMuxLocked() {
-	mux := http.NewServeMux()
+	newMux := mux.NewRouter().StrictSlash(true)
```

Comment:
well, not worse than the disease.  Really anything is better than what we have now, but it seems worse than telling people, "we should never have done this, find the right URL.  Here, we'll help"